pkg/v1/types/types.go

```diff
@@ -24,7 +24,9 @@ const (
 	OCIManifestSchema1             MediaType = "application/vnd.oci.image.manifest.v1+json"
 	OCIConfigJSON                  MediaType = "application/vnd.oci.image.config.v1+json"
 	OCILayer                       MediaType = "application/vnd.oci.image.layer.v1.tar+gzip"
+	OCILayerZStd                   MediaType = "application/vnd.oci.image.layer.v1.tar+zstd"
 	OCIRestrictedLayer             MediaType = "application/vnd.oci.image.layer.nondistributable.v1.tar+gzip"
+	OCIRestrictedLayerZStd         MediaType = "application/vnd.oci.image.layer.nondistributable.v1.tar+zstd"
```

Comment:
I'd prefer not to add this for now until we have a use case for it.

We might be able to get rid of non-distributable layers entirely: https://github.com/opencontainers/image-spec/pull/965

Reply:
Ok, that makes sense. I'll remove it.